app/helpers/timelines_helper.rb

```diff
@@ -182,4 +182,37 @@ def push_timeline_options(timeline, target = gon)
   def timeline_options
     OpenStruct.new @timeline.options
   end
+
+  def new_timeline_link(project, &_block)
```

Comment:
Why are you including the block parameter if you are not using it anyway?

I checked my ruby-foo against this neat article and it tells me what I thought to know: You only need the block-parameter if you want toget fancy: http://blog.codahale.com/2005/11/24/a-ruby-howto-writing-a-method-that-uses-code-blocks/